metadata-load/pom.xml

```diff
@@ -81,7 +76,7 @@
                     <plugin>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <excludedGroups>uk.ac.ebi.ampt2d.metadata.loader.database.OracleDbCategory</excludedGroups>
+                            <excludedGroups>uk.ac.ebi.ampt2d.metadata.pipeline.loader.database.OracleDbCategory</excludedGroups>
```

Comment:
I personally think the simplest reasonable package structure would be:

uk.ac.ebi.ampt2d.metadata
|- importer
...|- configuration
...|- api
...|- converter
...|- ...

This way instead of 'pipeline' and 'load', which are a very generic terms, we have 'import' because this pipeline imports data from ENA. And also we don't add more (almost empty) package levels.